src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -1130,58 +1138,77 @@ private static void setCommitLogUpperBound(AtomicReference<ReplayPosition> commi
      * Finds the largest memtable, as a percentage of *either* on- or off-heap memory limits, and immediately
      * queues it for flushing. If the memtable selected is flushed before this completes, no work is done.
      */
-    public static class FlushLargestColumnFamily implements Runnable
+    public static CompletableFuture<Boolean> flushLargestColumnFamily()
```

Comment:
@ekaterinadimitrova2 So we no longer have to worry about the `minOwnershipRatio`, but we do keep the `CompletableFuture` concept so that we can properly update the metric?

Reply:
Naming nit stands: https://github.com/ekaterinadimitrova2/cassandra/pull/74/files?file-filters%5B%5D=.java#r534677784

Reply:
I agree but I wasn't sure that it will be appreciated to change an approved already naming convention from another patch.